graphistry/embed_utils.py

```diff
@@ -21,12 +21,12 @@ def lazy_embed_import_dep():
     except:
         return False, None, None, None, None, None, None, None
 
-# def check_cudf():
-#     try:
-#         import cudf
-#         return True, cudf
-#     except:
-#         return False, object
+def check_cudf():
+    try:
+        import cudf
```

Comment:
1. Who calls this on import?

2. And can this be a a) cached call that b) checks module path vs an import?

Reply:
its only `test_embed_utils` that calls `check_cudf`, swapped out for `lazy_cudf_import` from `umap_utils`

Reply:
oh, so that shouldn't be the issue, right? `test/*` shouldn't get imported by `import graphistry`..

Reply:
sorry, no i wasnt clear, `test_embed_utils` is only **OTHER** place `lazy_cudf_import` was present. It was used in `embed_utils` and imported cudf to check df dtype, but I have swapped it out in place of just checking via getmodule _e.g._ `if 'cudf' in str(getmodule(self._nodes)):` , so I believe the problem is solved -- tuna looks much better
<img width="1138" alt="Screenshot 2023-08-01 at 10 44 04" src="https://github.com/graphistry/pygraphistry/assets/1395951/706cfbab-c74a-4016-8ea8-20b828abbf17">